Sources/eppo/dto/EppoValue.swift

```diff
@@ -188,3 +188,20 @@ public class EppoValue : Decodable, Equatable {
         }
     }
 }
+extension EppoValue {
```

Comment:
Is this to support the `Codable` interface?

Reply:
Yes